polyply/src/top_parser.py

```diff
@@ -392,9 +393,29 @@ def _molecule(self, line, lineno=0):
         """
         self.current_itp.append(line)
 
+    def parse_error(self, line, lineno=0):
+        """
+        Parse the #error statement.
+        """
+        if self.current_meta:
+           # the #error is between ifdef
+           # however tag is not in defines we have
+           # read so the error is not triggered.
+           if self.current_meta["condition"] == "ifdef"\
+              and self.current_meta["tag"] not in self.topology.defines:
+                 return
+           # the current file is between ifndef
+           # so if tag is defined we ignore this file
+           elif self.current_meta["condition"] == "ifndef"\
+              and self.current_meta["tag"] in self.topology.defines:
+                 return
+        # we remove the #error
+        msg = line[7:]
+        raise NotImplementedError(msg)
```

Comment:
Is this the right behaviour? And if it is, is it the right exception type? Why not IOError?